h/static/styles/common.scss

```diff
@@ -5,6 +5,7 @@
 @import 'mixins/responsive';
 @import 'grid';
 @import 'annotations';
+@import 'excerpt';
```

Comment:
This should go under 'components' in `app.scss`, assuming it isn't used by the help page.